### DISCUSSION TITLE
Whisper recommendation

### DISCUSSION THREAD

**Discussion by wriede** [justLV/onju-voice#24] [Q&A]
Can I ask what computer hardware people are using to run Whisper on as is needed for these modified Minis?

**Comment by LordNex** [comment]
Home Assistant with the Wyoming Protocol installed. If you want conversational you'll probably need a ChatGPT account and pinecone 

Although I'm still just using my mini's in there normal state. Haven't swapped the boards yet. And I've had limited results with RaspiAudio MuseLUX as well which definitely doesn't look better on the wall.
